### PR TITLE
Create concat.yaml

### PR DESCRIPTION
**Pull request by PeterKraus**

### CODE REVIEW

File: src/instances/concat.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+id: concat
+name: >-
+    ConCat lab at TU Berlin
+contact:
+    - name: Peter Kraus
+      email: peter.kraus@tu-berlin.de
+canonical_url: https://datalab.concatlab.eu
+api_url: https://datalab-api.concatlab.eu
```
Comment (ml-evs) [inline src/instances/concat.yml:9]:
```suggestion
api_url: https://datalab-api.concatlab.eu
homepage: https://www.tu.berlin/en/concat
```

For some free advertising perhaps?

**Review by ml-evs — CHANGES_REQUESTED** [review]
I assume you purposefully made a .yml rather than .yaml to make a point? wonder if there's a pre-commit rule I can add for that

**Comment by PeterKraus** [comment]
I have to say I wasn't really thinking, I just copied and pasted the text from one of the other files and got the file name wrong.

**Review by ml-evs — APPROVED** [review]
:rocket:

**Pull request merged**